packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/styles.ts

```diff
@@ -66,6 +67,14 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
       }),
       autoprefixer(),
     ];
+
+    if (wco.buildOptions.optimization.styles) {
+      postCssPlugins.push(
+        cssNano({ preset: 'default' }),
```

Comment:
This will operate differently than the current setup.  Each input file will be optimized in isolation whereas with the previous approach the final output was optimized as a whole.  The later has advantages in terms of both performance and file size reduction.